schedule/Schedule_models.py

```diff
@@ -0,0 +1,74 @@
+from django.db import models
+from django.contrib.postgres.fields import ArrayField
+from itertools import chain
+from django.utils.translation import gettext_lazy as _
+
+
+'''PRIMARY KEY: dayOfWeek + timeRange'''
+class A_TimeSlot(models.Model):
```

Comment:
the A_ prefix just indicates this is for algorithms correct?

Reply:
That's right, I thought it was a clean naming convention to avoid conflicting with FE-side models, like `Course`